keygen/src/keygen.rs

```diff
@@ -281,6 +281,13 @@ fn app<'a>(num_threads: &'a str, crate_version: &'a str) -> Command<'a> {
                 .arg(
                     derivation_path_arg()
                 )
+                .arg(
+                    Arg::new("bs58_keypair_string")
+                    .value_name("BS58_KEYPAIR_STRING")
+                    .takes_value(true)
+                    .long("bs58-keypair-string")
+                    .help("Base58 encoded keypair string")
+                )
```

Comment:
nit: I think let's be a pretty obvious on the CLI argument and use `base58` rather than `bs58`.